encoding/json/json.go

```diff
@@ -0,0 +1,23 @@
+//go:build !sonic
+
+package json
+
+import "encoding/json" //nolint:depguard // This is a wrapper package for encoding/json. It serves as the default JSON package for GCT (GoCryptoTrader). All uses of JSON throughout the application should refer to this package with the default build.
+
+// Version is a constant string that represents the current JSON implementation package
+const Version = "encoding/json"
```

Comment:
I don't think Version is the right term for this.

Implementation maybe.

However I don't see any reason to need to know apart from logging it, which I'm about to recommend removing :)

Reply:
I use this externally to confirm whether Sonic is in use, as I often mistype build tags. I’d prefer keeping it. I can change to implementation.

Reply:
I requested it to have an in-application acknowledgement of how it was built since that can happen as a seperate step. Kinda like how we log runtime args too. Can be helpful on log history.